lib/Widgets/AbstractSimpleSettingsPage.vala

```diff
@@ -87,27 +90,27 @@ public abstract class Granite.SimpleSettingsPage : Granite.SettingsPage {
         }
 
         if (activatable) {
-            status_switch = new Gtk.Switch ();
-            status_switch.hexpand = true;
-            status_switch.halign = Gtk.Align.END;
-            status_switch.valign = Gtk.Align.CENTER;
+            status_switch = new Gtk.Switch () {
+                hexpand = true,
+                halign = Gtk.Align.END,
+                valign = Gtk.Align.CENTER
+            };
             header_area.attach (status_switch, 2, 0);
         }
 
-        content_area = new Gtk.Grid ();
-        content_area.column_spacing = 12;
-        content_area.row_spacing = 12;
-        content_area.vexpand = true;
-
-        action_area = new Gtk.Box (Gtk.Orientation.HORIZONTAL, 6);
+        content_area = new Gtk.Grid () {
+            column_spacing = 12,
+            row_spacing = 12,
+            vexpand = true
+        };
+        content_area.add_css_class ("content-area");
```

Comment:
Does the `content-area` style class set up the spacing? The header_area seems to remove the hard coded spacing and places in the css style. Not sure where these classes come from (can't find it in Granite or the eOS stylesheet) so I am not sure where to look to answer this myself.

Reply:
It’s currently unused, but I wanted to make sure app developers had an easy way to select it